app/coffeescripts/jquery/ModuleSequenceFooter.coffee

```diff
@@ -131,6 +134,10 @@ define [
         @buildNextData() if (@next.show = @item.next)
         @buildPreviousData() if (@previous.show = @item.prev)
 
+        # Make progress information available
+        @items_count = @modules[0].items_count
```

Comment:
On Fri, Jul 17, 2015 at 11:01:56AM -0700, Brian Sadler wrote:
> Are you sure we want modules[0] ?  Is that just the current module?

Yeah, from what I can tell at least (I checked the api json
on staging in several cases and it was consistently what we needed.
Most things didn't even have other entries in that array, I believe it
is an array for the cases where the same page appears on multiple modules.)